circuits/circom/updateStateTree.circom

```diff
@@ -382,7 +387,7 @@ template UpdateStateTree(
     valid_state_leaf_index.in[0] <== decrypted_command_out[CMD_STATE_TREE_INDEX_IDX];
     valid_state_leaf_index.in[1] <== state_tree_max_leaf_index;
 
-    component valid_vote_options_leaf_index = LessEqThan(8);
+    component valid_vote_options_leaf_index = LessEqThan(32);
```

Comment:
Does this mean 2^32?

Reply:
I think so. `LessThan(n)` checks the `n`th bit of the output of a `Num2Bits`: https://github.com/iden3/circomlib/blob/master/circuits/comparators.circom#L88

Is `2 ** n` the maximum value for either of the input signals into `LessEqThan`?